src/js/multiDomainFirstParties.js

```diff
@@ -12,8 +12,31 @@ var multiDomainFirstPartiesArray = [
   ["accountonline.com", "citi.com", "citibank.com", "citicards.com", "citibankonline.com"],
   ["allstate.com", "myallstate.com"],
   ["altra.org", "altraonline.org"],
-  ["amazon.com", "amazon.com.au", "amazon.ca", "amazon.co.jp", "amazon.co.uk", "amazon.de",
-    "amazon.es", "amazon.fr", "amazon.it", "ssl-images-amazon.com", "media-amazon.com"],
+  [
+    "amazon.com",
+
+    "amazon.ca",
+    "amazon.co.jp",
+    "amazon.com.au",
+    "amazon.com.mx",
+    "amazon.co.uk",
+    "amazon.de",
+    "amazon.es",
+    "amazon.fr",
+    "amazon.in",
+    "amazon.it",
+
+    "6pm.com",
+    "audible.com",
```

Comment:
Should we add audible.co.uk, audible.de?

Reply:
Yep, good catch, will do so now.

Reply:
Added and rebased on master.